source/kraken/maintenance_worker.cpp

```diff
@@ -172,8 +172,7 @@ void MaintenanceWorker::handle_rt_in_batch(const std::vector<AmqpClient::Envelop
         for(const auto& entity: feed_message.entity()){
             if (!data) {
                 data = data_manager.get_data_clone();
-                data->last_rt_data_loaded = pt::microsec_clock::universal_time();
-                LOG4CPLUS_INFO(logger, "data copied in " << (data->last_rt_data_loaded - begin));
+                LOG4CPLUS_INFO(logger, "data copied in " << (pt::microsec_clock::universal_time() - begin));
```

Comment:
can you create/use an other "begin" (to avoid counting the parsing in the copy) ?